### PR TITLE
file upload method added

### PR DESCRIPTION
**Pull request by keerthu**

### CODE REVIEW

**Comment by keerthu** [comment]
merged

File: spotify/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+
+            if (!sb.toString().trim().isEmpty()) {
+                out = sb.toString();
+            }
+        }
+
+        OMElement omElement = AXIOMUtil.stringToOM(out);
+
+        return omElement;
+
```
Comment (prabathabey) [inline spotify/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java]: Unnecessary new line.

File: spotify/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+            File connectorPropertyFile = new File(connectorConfigFile);
+            InputStream inputStream = null;
+            if (connectorPropertyFile.exists()) {
+                inputStream = new FileInputStream(connectorPropertyFile);
+            }
+
+            if (inputStream != null) {
+                Properties prop = new Properties();
+                prop.load(inputStream);
+                inputStream.close();
```
Comment (prabathabey) [inline spotify/src/test/java/org/wso2/carbon/connector/integration/test/common/ConnectorIntegrationUtil.java]: inputStream not closed in a finally block which can lead to potential memory leaks!

File: spotify/src/main/java/org/wso2/carbon/connector/spotify.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+package org.wso2.carbon.connector;
+
+import org.apache.synapse.MessageContext;
+import org.wso2.carbon.connector.core.*;
+
+public class spotify extends AbstractConnector {
```
Comment (lafernando) [inline spotify/src/main/java/org/wso2/carbon/connector/spotify.java]: Is this some class created for testing? .. anyways, never name a Java class starting with a lower case character ..

**Pull request merged**